pkg/store/proxy_heap_test.go

```diff
@@ -22,6 +22,132 @@ func TestRmLabelsCornerCases(t *testing.T) {
 	}), labels.Labels{})
 }
 
+type testingRespSet struct {
```

Comment:
Can we reuse the mocks and stubs from the `testutil` package? You can see an example usage [here](https://github.com/thanos-io/thanos/blob/main/pkg/store/proxy_test.go#L79).

Reply:
I could not find an implementation of `respSet` in `testutil`, but I changed it to use one from proxy_head.go.